### PR TITLE
EPAS - Added content as per DF-285 for EPAS 10 to 14 versions

### PR DESCRIPTION
**Pull request by nidhibhammar**

### CODE REVIEW

File: product_docs/docs/epas/14/epas_inst_linux/managing_an_advanced_server_installation.mdx [hunk 1]
```diff
@@ -285,7 +285,7 @@ In addition to the cluster configuration options documented in the PostgreSQL co
 
 `--no-redwood-compat`
 
-Include the `--no-redwood-compat` keywords to instruct the server to create the cluster in PostgreSQL mode. When the cluster is created in PostgreSQL mode, the name of the database superuser is `postgres`, the name of the default database is `postgres`, and EDB Postgres Advanced Server’s features compatible with Oracle databases aren't available to the cluster.
+Include the `--no-redwood-compat` keywords to instruct the server to create the cluster in PostgreSQL mode. When the cluster is created in PostgreSQL mode, the name of the database superuser is `postgres`, the name of the default database is `postgres`. The few Advanced Server’s features compatible with Oracle databases will be available with this mode. However, we recommend using the Advanced server in redwood compatibility mode to use all its features.
```
Comment (moiznalwalla) [inline product_docs/docs/epas/14/epas_inst_linux/managing_an_advanced_server_installation.mdx:288]: might be worth checking with Betsy, but it seems to me like there should be an "and" before "the name of the default database is `postgres`."

**Review by moiznalwalla — CHANGES_REQUESTED** [review]
Added a small comment

File: product_docs/docs/epas/14/epas_inst_linux/managing_an_advanced_server_installation.mdx (hunk 1 above)
Comment (moiznalwalla) [reply product_docs/docs/epas/14/epas_inst_linux/managing_an_advanced_server_installation.mdx]: Does this sound better in stead of the last two lines?

"Not all Oracle-compatible features in Advanced Server are available with this mode. We recommend using the Advanced server in redwood compatibility mode to use all the Oracle-compatible features."

Comment (nidhibhammar) [reply product_docs/docs/epas/14/epas_inst_linux/managing_an_advanced_server_installation.mdx]: I have added "and"

**Review by moiznalwalla — APPROVED** [review]

File: product_docs/docs/epas/14/epas_inst_windows/installing_advanced_server_with_the_interactive_installer/index.mdx [hunk 2]
```diff
@@ -54,7 +54,7 @@ In addition to the cluster configuration options documented in the PostgreSQL co
 
 `--no-redwood-compat`
 
-Include the `--no-redwood-compat` keywords to instruct the server to create the cluster in PostgreSQL mode. When the cluster is created in PostgreSQL mode, the name of the database superuser is `postgres`, the name of the default database is `postgres`, and EDB Postgres Advanced Server’s features compatible with Oracle databases aren't available to the cluster.
+Include the `--no-redwood-compat` keywords to instruct the server to create the cluster in PostgreSQL mode. When the cluster is created in PostgreSQL mode, the name of the database superuser is `postgres` and the name of the default database is `postgres`. The few Advanced Server’s features compatible with Oracle databases will be available with this mode. However, we recommend using the Advanced server in redwood compatibility mode to use all its features.
```
Comment (drothery-edb) [inline product_docs/docs/epas/14/epas_inst_windows/installing_advanced_server_with_the_interactive_installer/index.mdx:57]:
```suggestion
Include the `--no-redwood-compat` keywords to instruct the server to create the cluster in PostgreSQL mode. When the cluster is created in PostgreSQL mode, the name of the database superuser is `postgres` and the name of the default database is `postgres`. A small subset of Advanced Server features compatible with Oracle databases are available with this mode. However, we recommend using Advanced Server in redwood compatibility mode to have access to all its features.
```

**Comment by drothery-edb** [comment]
@ccestes , can you backport Nidhi's latest commit to the earlier versions? Also, let's clean up the convention for describing the init options in the topic for all versions too. Instead of using this convention:

Include the `--no-redwood-compat` keywords to instruct the ...

Let's use this for all the option descriptions.
`--no-redwood-compat` instructs the ...

**Comment by drothery-edb** [comment]
@ccestes I think you may have missed cleaning up the option descriptions in the v14 topic.

**Comment by nidhibhammar** [comment]
changes looks good to me.

**Comment by nidhibhammar** [comment]
@drothery-edb, this can be merged and published too

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**